src/witness/witness_calculator.rs

```diff
@@ -16,8 +16,10 @@ use super::Circom;
 
 #[derive(Clone, Debug)]
 pub struct WitnessCalculator {
-    pub instance: Wasm,
+    pub store: Arc<RwLock<Store>>,
+    pub instance: WasmInstance,
     pub memory: SafeMemory,
+    // Number of 64-bit blocks required to represent a field element?
```

Comment:
?

Reply:
if so, maybe `num_limbs` is better?

Reply:
I just updated the `Memory` and `WitnessCalculator` variable names to reduce ambiguity of limbs (since Memory uses 32-bit limbs)